ocaml/xapi/xapi_mgmt_iface.ml

```diff
@@ -45,7 +45,7 @@ let restart_stunnel_nomutex ~__context ~accept =
     then [ "back_compat_6_5" ]
     else []
   in
-  let xapissl_args = [ "restart"; accept ] @ (back_compat ~__context) in
+  let xapissl_args = [ "restart"; accept ] @ (back_compat ~__context) @ ["permfile=" ^ !Xapi_globs.server_cert_path] in
```

Comment:
I like to suggest to use `List.concat` - it has better performace and it makes it easier to use function calls to construct argument lists. A vertical layout is good for future additions.

```
let xapissl_args = List.concat
  [ [ "restart"; accept ]
  ;  back_compat ~__context
  ; ["permfile=" ^ !Xapi_globs.server_cert_path]
  ]
```